src/components/incident/SignOffAction.tsx

```diff
@@ -116,11 +116,6 @@ const SignOffAction: React.FC<SignOffActionPropsType> = ({
             id="message"
             label={dialogInputLabel}
             type={dialogInputType}
-            inputProps={
-              dialogInputType === "url" ?
-                { pattern: "^(http(s)?:\\/\\/)+[\\w\\-\\._~:\\/?#[\\]@!\\$&'\\(\\)\\*\\+,;=.]+$" } :
-                {}
-            }
```

Comment:
Good, duplicate regular expressions sprinkled through the code is a BIG code smell.